autoload/codi/load.vim

```diff
@@ -67,6 +67,10 @@ let s:codi_default_interpreters = {
           \ 'bin': ['planck', '--verbose', '--dumb-terminal'],
           \ 'prompt': '^.\{-}=> ',
           \ },
+      \ 'php': {
+          \ 'bin': ['psysh'],
+          \ 'prompt': '^\(>>>\|\.\.\.\) ',
```

Comment:
Can you add a `preprocess` to get rid of the fat arrow from `psysh` output? It should look pretty much the same as `s:pp_r`. In fact, you might just want to rename the function to `s:pp_fat_arrow` and reuse it.